src/manage_pg.c

```diff
@@ -3263,17 +3267,17 @@ manage_db_init_indexes (const gchar *name)
            " ON scap2.cves (creation_time);");
       sql ("CREATE INDEX cves_by_modification_time_idx"
            " ON scap2.cves (modification_time);");
-      sql ("CREATE INDEX cves_by_score"
-           " ON scap2.cves (score);");
+      sql ("CREATE INDEX cves_by_severity"
+           " ON scap2.cves (severity);");
 
       sql ("CREATE UNIQUE INDEX cpe_idx"
            " ON scap2.cpes (name);");
       sql ("CREATE INDEX cpes_by_creation_time_idx"
            " ON scap2.cpes (creation_time);");
       sql ("CREATE INDEX cpes_by_modification_time_idx"
            " ON scap2.cpes (modification_time);");
-      sql ("CREATE INDEX cpes_by_score"
-           " ON scap2.cpes (score);");
```

Comment:
What about removing the existing now obsolete indexes? Do we need to remove them in a migration?

Reply:
For the SCAP and CERT data the schema is fully rebuilt if the version changes, so this should not be necessary.